include/sensors/core/tasks/pressure_driver.hpp

```diff
@@ -322,7 +320,8 @@ class MMR920C04 {
     uint16_t number_of_reads = 0x1;
     // TODO(fs, 2022-11-11): Need to figure out a realistic threshold. Pretty
     // sure this is an arbitrarily large number to enable continuous reads.
-    float threshold_cmH20 = 2457.0F;
+    float threshold_pascals = 100.0F;
+    float pressure_pascals = 0.0F;
```

Comment:
I don't think we should have this be a private variable on the pressure driver class.